rebasehelper/tests/test_specfile.py

```diff
@@ -231,6 +231,8 @@ def test_get_extra_version_set(self, spec_object):
         assert spec_object.get_extra_version() == 'rc1'
 
     def test_update_setup_dirname(self, spec_object):
+        spec_object.set_extra_version('rc1')
```

Comment:
Wouldn't it make more sense to set extra version below line `assert spec_object.spec_content.section('%prep') == prep` since extra version doesn't influence the first call of `update_setup_dirname`?

Reply:
I don't think it would, this is basically part of setup phase of this test, and as such, it should be at the beginning.